tensorflow/lite/micro/tools/ci_build/test_cortex_m_corstone_300.sh

```diff
@@ -51,4 +51,5 @@ readable_run make -j$(nproc) -f tensorflow/lite/micro/tools/make/Makefile \
   TARGET_ARCH=${TARGET_ARCH} \
```

Comment:
The `CO_PROCESSOR=ethos_u` line needs to be removed.  With the co-processor specified, the simulator does not make the instruction cycle counter available.  I have tested this locally.

Reply:
cpu cycle count on fvp is not accurate, and is not good for benchmarking! Should be able to get the count with ethos u as that is what the fvp was made to be cycle accurate to

Reply:
Understood, however I just get zeros from the `GetCurrentTimeTicks` function with the co-processor defined.

Reply:
Please refer to this - [run-on-fvp-based-on-arm-corstone-300-software]( https://github.com/tensorflow/tflite-micro/blob/main/tensorflow/lite/micro/benchmarks/README.md#run-on-fvp-based-on-arm-corstone-300-software ) - for why  only `CO_PROCESSOR=ethos_u` makes sense when running on Corstone-300 FVP and you care about cycle counter.
However supplying` CO_PROCESSOR=ethos_u` is not enough. The supplied tflite model must first be optimized so it contains the Ethos-U custom op. Within the examples that comes with tflite-micro there are only one such example and that is the person detect model that can be run with the network tester example or the benchmarks.
So the rationale for test_cortex_m_corstone_300.sh is that here we don't care about cycle measurements. We could then remove `CO_PROCESSOR=ethos_u.` However the rationale for keeping is to compile test that code path.

Reply:
If I understand correctly:

* If we have `CO_PROCESSOR=ethos_u` defined, we'll read the NPU's cycle counts for operations.
* The model we're using for test here doesn't have the ethos_u custom op in the model, so regardless of `CO_PROCESSOR`, we'll be executing on CPU.

The way I see it, we have three options (feel free to suggest alternatives):
1. Use the modified person detect model with Ethos-U custom op and enable co-processor. We should get proper benchmark numbers for person detect on Ethos-U.
2. Keep test as is, where we're running person detect on CPU and reading the inaccurate CPU cycle counts. Add a disclaimer that these are not accurate results and should not be relied upon.
3. Remove the test to not be misleading. Downside here is we won't have continuous testing of tflm_benchmark utility on ARM, but maybe that's okay if we can't trust it.

I would lean towards 1 or 2. @mansnils @ArmRyan WDYT?